### PR TITLE
fix YouTube playlist title on Resources Page

### PR DESCRIPTION
**Pull request by michow**
## What this PR does / why we need it
Fix typo on Resources page. YouTube series is spelled Enlightning without the "e".

## Which issue(s) this PR fixes

Fixes: #

### CODE REVIEW

File: docs/site/themes/template/layouts/_default/resources.html [hunk 1]
```diff
@@ -121,7 +121,7 @@ <h2>Additional resources</h2>
                         <div class="panel" id="panelAdditional3" aria-labelledby="additional3" role="region">
                             <br />
                             <div class="subpanel">
-                                <h3>Enlightening</h3>
+                                <h3>Enlightning</h3>
```
Comment (stmcginnis) [inline docs/site/themes/template/layouts/_default/resources.html:124]: The current spelling is actually correct.

https://www.merriam-webster.com/dictionary/enlightening

**Review by stmcginnis — CHANGES_REQUESTED** [review]

**Comment by michow** [comment]
@jpmcb @stmcginnis @jkotzen I do not have the permissions to reopen this pull request, someone from your side will have to do it. The name of the YouTube series referred to on the Resources page is indeed spelled "Enlightning" without the additional "e".

**Comment by stmcginnis** [comment]
Please update the title and description to actually reflect what is being done and why. Thanks!

**Comment by michow** [comment]
@stmcginnis @jpmcb title and description have been updated

**Review by stmcginnis — APPROVED** [review]
Thanks!

**Pull request merged**